README.md

```diff
@@ -407,6 +407,19 @@ These settings can be changed at any time.
   Supported values are `debug`, `info`, `warn`, `error`, and `crit`, and the default is `info`.
 * `settings.ecs.enable-spot-instance-draining`: If the instance receives a spot termination notice, the agent will set the instance's state to `DRAINING`, so the workload can be moved gracefully before the instance is removed. Defaults to `false`.
 
+#### Image registry settings
```

Comment:
We talk about OS images in this readme too, so it might be worthwhile to call out that this is specific to container images.

```suggestion
#### Container image registry settings
```